pfSense-pkg-API/files/etc/inc/api/models/APIFirewallRuleCreate.inc

```diff
@@ -179,8 +182,8 @@ class APIFirewallRuleCreate extends APIModel {
 
 
         # Check for our optional 'disabled' payload value
-        if ($this->initial_data['disabled'] !== true) {
-            $this->validated_data["id"] = "";
+        if ($this->initial_data['disabled'] === true) {
```

Comment:
For this one I can't. `===` is the only way to do strict type matching in PHP. Otherwise if a client passes `{"disabled": "no"}` in their payload it will match the statement and the rule will be disabled.